docs/FullGuide.md

```diff
@@ -258,8 +319,8 @@ Note that registering this callback will enable callbacks for both data and erro
 
 Your callback can implement the following methods: 
 
-* `onErrorBeforeClose(stream, error)` - called when the stream has been stopped but not yet closed, so you can still query the stream for its properties (e.g. for number of underruns). You can also inform any other threads that may be calling the stream to stop doing so.
-* `onErrorAfterClose(stream, error)` - called when the stream has been closed by Oboe so the stream cannot be used and calling getState() will return closed. 
+* `onErrorBeforeClose(stream, error)` - called when the stream has been stopped but not yet closed, so you can still query the stream for its properties (e.g. for number of underruns). You can also inform any other threads that may be calling the stream to stop doing so. Do not delete the stream or modify its stream state in this callback.
+* `onErrorAfterClose(stream, error)` - called when the stream has been closed by Oboe so the stream cannot be used and calling getState() will return closed. The only valid queries are frames written and read. The stream itself can safely destroy itself at the end of this method (although this is risky). Triggering another stream to restart is also a valid use of this callback.
```

Comment:
We should have clearer guidance here, not a fan of "although this is risky". Maybe something like "At this point the stream has been destroyed and you can safely start a new one".

Reply:
The callback can delete the stream. It can also start another stream. And can query underruns

Reply:
Everything changes after disconnect